### PR TITLE
Update README to be more easy to read and understand.

### PR DESCRIPTION
**Pull request by spirillen**
Have done my best to rewrite this to a more softly text user friendly and easy to read and understand.

Also fixed various typos... according to my spellchecker.

End note:
There are still phrases which isn't optimal, but I do not currently have better suggestions.

/spend 1h51m

To my self:
image

### CODE REVIEW

**Comment by smed79** [comment]
@funilrys @mitchellkrogza

**Comment by spirillen** [comment]
@smed79 What is your own opinion on the rewrite?

**Comment by smed79** [comment]
> What is your own opinion on the rewrite?

No opinion (_English is my third language and very limited_).

**Comment by spirillen** [comment]
> No opinion (_English is my third language and very limited_).

Then you absolutely should have an opinion :smiley: 

Do you fell it is easier to understand and read and the original text?

File: README.rst [hunk 1]
```diff
@@ -22,56 +23,74 @@ Installation
...
 ---------------
 
-If you already used a whitelist list you already know that we generaly only list all domains we want to whitelist one by one.
+If you already have tried to use a whitelist, you'll probably know, that in 
+generally you can only add one domain or URL per line in a file, for which you 
+want to whitelist.
 
-It's also possible to do that with our whitelisting system but we can do more.
+With UHBW you can do this, but in addition to that tedious way, UHBW allows you 
+to use ``Regex``, :code:`RZD` and :code:`ALL`
```
Comment (funilrys) [inline README.rst]: @spirillen it's 

```
:code:`REG`
```

File: README.rst [hunk 2]
```diff
@@ -80,73 +99,86 @@ If we have the following secondary whitelist list:
...
 
     usage: uhb_whitelist [-h] [-a ANTI_WHITELIST [ANTI_WHITELIST ...]] [-d]
-                        [-f FILE] [-o OUTPUT] [-m] [-p PROCESSES] [-v]
-                        [-w WHITELIST [WHITELIST ...]] [-wc]
+			 [-f FILE] [-o OUTPUT] [-m] [-p PROCESSES] [-v]
+                         [-w WHITELIST [WHITELIST ...]] [-wc]
 
-    The tool to clean a list or a hosts file with the Ultimate Hosts Blacklist
-    whitelist list or your own.
+    UHBW is a tool to clean up lists or hosts files with the hosted and/or your
```
Comment (funilrys) [inline README.rst]: @spirillen Please change in the source code this part. ==> https://github.com/Ultimate-Hosts-Blacklist/whitelist/blob/4b7343eb25f359a1f2693763238ae89157751681/ultimate_hosts_blacklist/whitelist/__init__.py#L169-L332

Comment (funilrys) [reply README.rst]: Clarification: It's generated by argparse (`--help`), it's just a script which updates this section. So changes should be made in the code itself.

File: README.rst (hunk 1 above)
Comment (spirillen) [reply README.rst]: I did think of that, but for a new reader, what is REG? so I left it intentionally as Regex for the initial reading to point the attention to this link Regex = REG

But if you prefer the other, I'll change.

File: README.rst (hunk 2 above)
Comment (spirillen) [reply README.rst]: Sure, but that isn't today ping me later if I forget it (I'm not that fresh today. Maybe I've someone from China :rofl: )

Comment (spirillen) [reply README.rst]: Submitted, what command would I use to generate a (pre-)build of this so I could verify the output locally? I most concerned about the layout in the `--help` output in a terminal

File: ultimate_hosts_blacklist/whitelist/__init__.py [hunk 3]
```diff
@@ -277,14 +278,16 @@ def _command_line():
...
         type=argparse.FileType("r"),
         nargs="+",
-        help="Read the given file and append its data to the our whitelist list.",
+        help="Read the given file and append its data to the UHBW's\n"
+        "hosted whitelist list.",
     )
 
     parser.add_argument(
-        "-wc",
+        "-wc\n",
```
Comment (funilrys) [inline ultimate_hosts_blacklist/whitelist/__init__.py]: @spirillen, linebreak (`\n`) is not authorized with `-wc`. By doing so, you are killing the whole switch!

File: ultimate_hosts_blacklist/whitelist/__init__.py [hunk 4]
```diff
@@ -277,14 +278,16 @@ def _command_line():
...
         "--without-core",
         action="store_false",
-        help="Disable the usage of the Ultimate Hosts Blacklist whitelist list.",
+        help="Disable the usage of the Ultimate Hosts Blacklist\n"
```
Comment (funilrys) [inline ultimate_hosts_blacklist/whitelist/__init__.py]: If linebreak is also needed from the CLI when you call `--help`, you need to implement what I did inside PyFunceble: 
 https://github.com/funilrys/PyFunceble/blob/d6033601909e88495f5b102e45ee7d5a48dbe46a/PyFunceble/cli/__init__.py#L122

File: README.rst (hunk 1 above)
Comment (funilrys) [reply README.rst]: Okay let's leave it like that!

File: ultimate_hosts_blacklist/whitelist/__init__.py (hunk 3 above)
Comment (spirillen) [reply ultimate_hosts_blacklist/whitelist/__init__.py]: f....

File: ultimate_hosts_blacklist/whitelist/__init__.py (hunk 4 above)
Comment (spirillen) [reply ultimate_hosts_blacklist/whitelist/__init__.py]: I leave this implementation to you @funilrys as I would not try to looks like I know anything about python :smile: which I don't

File: ultimate_hosts_blacklist/whitelist/__init__.py (hunk 3 above)
Comment (spirillen) [reply ultimate_hosts_blacklist/whitelist/__init__.py]: Believes I've cleaned up those I had made. See also keybase

**Pull request merged**